app/views/onboarding/OnboardingNotifications.tsx

```diff
@@ -1,49 +1,126 @@
 import React, { useContext } from 'react';
+import {
+  ImageBackground,
+  View,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+import { useTranslation } from 'react-i18next';
+import { useNavigation } from '@react-navigation/native';
+import { SvgXml } from 'react-native-svg';
+
 import PermissionsContext from '../../gps/PermissionsContext';
+import { Screens } from '../../navigation';
+import { Typography } from '../../components/Typography';
+import { useStatusBarEffect } from '../../navigation';
 
 import { Icons, Images } from '../../assets';
-
-import OnboardingTemplate from './OnboardingTemplate';
-import { useTranslation } from 'react-i18next';
 import {
-  NavigationParams,
-  NavigationScreenProp,
-  NavigationState,
-} from 'react-navigation';
-import { Screens } from '../../navigation';
-
-type OnboardingNotificationsProps = {
-  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
-};
+  Buttons,
+  Spacing,
+  Colors,
+  Iconography,
+  Typography as TypographyStyles,
+} from '../../styles';
 
-const OnboardingNotifications = ({
-  navigation,
-}: OnboardingNotificationsProps): JSX.Element => {
+const OnboardingNotifications = (): JSX.Element => {
+  const navigation = useNavigation();
+  const { t } = useTranslation();
   const { notification } = useContext(PermissionsContext);
 
-  const requestNotifications = async () => {
+  useStatusBarEffect('dark-content');
+
+  const requestPermission = async () => {
     await notification.request();
-    continueOnboarding();
   };
 
-  const continueOnboarding = () =>
+  const continueOnboarding = () => {
     navigation.navigate(Screens.OnboardingLocationPermissions);
+  };
 
-  const { t } = useTranslation();
+  const handleOnPressEnable = async () => {
+    await requestPermission();
+    continueOnboarding();
+  };
+
+  const handleOnPressMaybeLater = () => {
+    continueOnboarding();
+  };
 
   return (
-    <OnboardingTemplate
```

Comment:
Why move away from template here?
Isn't the only thing different for these screens:

1. the color (light vs. dark theme)
2. having a tertiary button? ("maybe later")

We keep wins on screen accessibility / layouts (something we've struggled to consistently maintain so far) by having a same template across our on boarding screens

Reply:
```We ended up needing to
remove the permissions screens from the purview of the
OnboardingTemplate as the permissions screens are significantly
different from the onboarding content screens in terms of markup, view
logic, and style so as to keep the onboarding template for them would
require introducing significant unnecessary complexity and ultimately
make the screens less amenable to future changes in requirements.```

A bit wordy, but i tried going down that route and it was a lot of effort to pull out that abstraction once the theme was gone.

In general trying to be overly dry with markup leads to views that are resistant to changes in future requirements. While its tempting to pull out abstractions based off of structure, over time you end up with these components that have dozens of conditionals, have multiple consumers all with slightly different needs, which leads to a situation where its more work thank necessary to update views to match new requirements.

 Its better to be dry in the data model and in the styles and maintain flexibility in the view.

Reply:
As for accessibility and layout, I agree we should be doing a better job with that, but that wont necessarily be addressed by putting template components everywhere. We still have to do the work of following best practices there.